### PR TITLE
Adagio: add user-sync config recommends

### PR DESCRIPTION
**Pull request by G15N**
## 🏷 Type of documentation

- [x] update bid adapter

## 📋 Checklist

- [ ] Related pull requests in prebid.js or server are linked -> Paste link in this list or reference it on the PR itself

### CODE REVIEW

File: dev-docs/bidders/adagio.md [hunk 1]
```diff
@@ -21,6 +21,24 @@ sidebarType: 1
...
+    filterSettings: {
+      iframe: {
+        bidders: '*',
+        filter: 'include'
+      }
+    }
+  }
+});
+```
+
```
Comment (osazos) [inline dev-docs/bidders/adagio.md]: What do think about the text below:

--- 

### Configuration

#### User Sync

Adagio strongly recommends enabling user syncing through iFrames. This functionality improves DSP user match rates and increases the bid rate and bid price. Make sure to call `pbjs.setConfig()` only once. This configuration is optional in Prebid, but required by Adagio.

```js
// https://docs.prebid.org/dev-docs/publisher-api-reference/setConfig.html#setConfig-Configure-User-Syncing
pbjs.setConfig({
  userSync: {
    filterSettings: {
      iframe: {
        bidders: ['adagio'],
        filter: 'include'
      }
    }
  }
});
```

File: dev-docs/bidders/adagio.md [hunk 2]
```diff
@@ -21,6 +21,26 @@ sidebarType: 1
 
 The Adagio bidder adapter requires setup and approval from the Adagio team. Please reach out to [contact@adagio.io](mailto:contact@adagio.io) for more information.
 
+### Configuration
+
+#### User Sync
+
+Adagio strongly recommends enabling user syncing through iFrames. This functionality improves DSP user match rates and increases the bid rate and bid price. Make sure to call pbjs.setConfig() only once. This configuration is optional in Prebid, but required by Adagio
```
Comment (osazos) [inline dev-docs/bidders/adagio.md]: pbjs.setConfig() must be displayed as code plz

Add a `.` at the end of the phrase.

File: dev-docs/bidders/adagio.md (hunk 1 above)
Comment (G15N) [reply dev-docs/bidders/adagio.md]: Fixed

File: dev-docs/bidders/adagio.md (hunk 2 above)
Comment (G15N) [reply dev-docs/bidders/adagio.md]: Fixed

**Review by osazos — APPROVED** [review]

**Pull request merged**